### DISCUSSION TITLE
Disabling the Cypress reporter panel in the browser

### DISCUSSION THREAD

**Discussion by codablock** [cypress-io/cypress#25513] [General]
Hello,

while trying to figure out why our Cypres based tests are running slow in CI, I stumbled accros the Cypress reporter panel (on the left side) taking quite some percentage of the CPU due to layouting.

It it possible to disable the reporter panel? I could not find anything in the documentation.

**Comment by ggrek1988** [comment]
@codablock Hi, Maybe you found a solution to your problem?

**Reply by codablock** [reply]
Unfortunately not. I also stopped working on the related project in the meantime.

**Comment by jennifer-shehane** [comment]
This feature will be available in v13. See [this PR](https://github.com/cypress-io/cypress/pull/27582) for more details.

Today there is some functionality of this by passing CYPRESS_NO_COMMAND_LOG=1 as an env var when you run and it hides the Command Log.
